testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveSparkCompatibility.java

```diff
@@ -97,4 +96,36 @@ public void testTrinoReadingTableCreatedByNativeSpark()
 
         onSpark().executeQuery("DROP TABLE " + sparkTableName);
     }
+
+    @Test(groups = {HIVE_SPARK_BUCKETING, PROFILE_SPECIFIC_TESTS})
```

Comment:
the HIVE_SPARK_BUCKETING group is wrong, since this test doesn't exercise spark bucketing

as a prep commit, rename HIVE_SPARK_BUCKETING group -> HIVE_SPARK